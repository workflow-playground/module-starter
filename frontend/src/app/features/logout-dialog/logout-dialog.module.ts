import { NgModule } from '@angular/core';
import { ModalModule, ButtonModule } from '@ds24/elements';
import { MatDialogModule } from '@angular/material/dialog';
import { TranslateModule } from '@ngx-translate/core';

import { LogoutDialogComponent } from './components/logout-dialog.component';

@NgModule({
  imports: [ModalModule, ButtonModule, MatDialogModule, TranslateModule],
  declarations: [LogoutDialogComponent],
})
export class LogoutFeatureModule {}
