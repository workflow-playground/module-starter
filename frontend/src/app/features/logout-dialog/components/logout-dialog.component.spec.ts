import { ComponentFixture, TestBed } from '@angular/core/testing';
import { LogoutDialogComponent } from './logout-dialog.component';
import { ButtonModule, ModalModule } from '@ds24/elements';
import { MatDialogModule } from '@angular/material/dialog';
import { TranslateModule } from '@ngx-translate/core';

describe('Logout Dialog Component', () => {
  let fixture: ComponentFixture<LogoutDialogComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      imports: [ModalModule, ButtonModule, MatDialogModule, TranslateModule.forRoot()],
      declarations: [LogoutDialogComponent],
    });

    fixture = TestBed.createComponent(LogoutDialogComponent);
  });

  it('should compile', () => {
    fixture.detectChanges();

    expect(fixture).toMatchSnapshot();
  });
});
