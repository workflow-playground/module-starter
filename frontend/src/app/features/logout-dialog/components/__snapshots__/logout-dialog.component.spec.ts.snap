// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout Dialog Component should compile 1`] = `
<ng-component>
  <ds-modal>
  </ds-modal>
</ng-component>
`;
